### DISCUSSION TITLE
Aerodynamic load calculation

### DISCUSSION THREAD

**Discussion by hefe2345** [OpenFAST/openfast#1455] [Q&A] [answered]
I want to calculate my blade's aerodynamic force with different wind velocities in the time domain. How can I use the Openfast code if I only want to calculate aerodynamic force due to different wind velocities? If AeroDyn is the tool, where can I find the user guide to pursue the task?

**Comment by ebranlard** [comment, marked as answer]
Hi @hefe2345 

You can change the wind velocity in the `InflowWind` input file. You can setup different OpenFAST simulations with different main `.fst` files, each pointing to different `InflowWind` input files. If you have to run many simulations, it can be tedious to setup all the input files manually, you can try to script it using the `python-toolbox` and the following example: 
https://github.com/OpenFAST/python-toolbox/blob/main/pyFAST/case_generation/examples/Example_Parametric.py

If you want to run simulations for a stiff structure, then the `aerodyn_driver` would be well suited, as it only uses `AeroDyn` and `InflowWind`.  You will have to compile the aerodyn driver yourself to obtain the executable.  

You'll find information about the aerodyn driver here:
https://openfast.readthedocs.io/en/dev/source/user/aerodyn/driver.html
Combined-case analyses can allow you to specify wind speeds into a table. 

Examples of aerodyn_driver input files can be found here:. 
https://github.com/OpenFAST/r-test/tree/main/modules/aerodyn
